src/App.js

```diff
@@ -1,11 +1,52 @@
 import GlobalStyle from "./constants/styles/global";
-import HomeScreen from "./screens/HomeScreen";
+import cantileverChair from "./assets/images/cantileverChair.png";
+import comfortHandy from "./assets/images/comfortHandyChair.png";
+import Navbar from "./components/nav";
+import FashionItemCard from "./components/cards/FashionItemCard";
+import image from "./assets/images/rect.png";
+import image2 from "./assets/images/rect-2.png";
+import ProductDetailsCard from "./components/cards/ProductDetailsCard";
+import bag from "./assets/images/bag.png";
+import bag1 from "./assets/images/bag1.png";
+import bag2 from "./assets/images/bag2.png";
+import bag3 from "./assets/images/bag3.png";
+import ProductHeroSection from "./components/cards/ProductHeroSection";
 
 function App() {
   return (
     <>
-      <HomeScreen />
+      {/* <HomeScreen /> */}
       <GlobalStyle />
```

Comment:
the GlobalStyle component should be the last component all others should be at the top of it.